### DISCUSSION TITLE
[@types/google-apps-script] Missing deprecation of file and folder fields for drive activity.

### DISCUSSION THREAD

**Discussion by azatoth** [DefinitelyTyped/DefinitelyTyped#66806] [Issues with a @types package]
While reading https://developers.google.com/drive/activity/v2/reference/rest/v2/activity/driveitem#DriveFile you can see that `file` and `folder` field has been deprecated, in favor of `driveFile` and `driveFolder`.

This is sadly missing from type definition herein:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/827a90da69a79a2cfdb56ed46a53daee596c3fb5/types/google-apps-script/apis/driveactivity_v2.d.ts#L79-L86

[a comment by an automated account is not shown]

**Comment by mtgto** [comment]
The type definition is out-of-date.
Update https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66810
